protos/orchestrator_service.proto

```diff
@@ -607,7 +607,8 @@ service TaskHubSidecarService {
 }
 
 message GetWorkItemsRequest {
-    // No parameters currently
+    int32 maxConcurrentOrchestrationWorkItems = 1;
```

Comment:
Let's use "orchestrator" instead of "orchestration" for work items, for consistency with our other existing code.
```suggestion
    int32 maxConcurrentOrchestratorWorkItems = 1;
```